spec/features/cases/offender_sar/case_creating_spec.rb

```diff
@@ -24,6 +24,22 @@
     then_expect_open_cases_page_to_be_correct
   end
 
+  scenario "0 Rejected offender SAR created after using back link" do
```

Comment:
You've got two "0" scenarios now. I'm not sure why they are numbered so you could update or remove the numbering.